spec/negotiation.html

```diff
@@ -353,7 +353,31 @@ <h1>
         1. Else,
           1. Assert: _type_ is ~string~.
           1. Set _value_ to ? ToString(_value_).
-        1. If _values_ is not ~empty~ and _values_ does not contain an element equal to _value_, throw a *RangeError* exception.
+        1. If _values_ is not ~empty~ and _values_ does not contain _value_, throw a *RangeError* exception.
+        1. Return _value_.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-getbooleanorstringnumberformatoption" type="abstract operation">
+      <h1>
+        GetBooleanOrStringNumberFormatOption (
```

Comment:
It was actually renamed _from_ GetStringOrBooleanOption at my request because the logic should not be generalized beyond NumberFormat; see https://github.com/tc39/proposal-intl-numberformat-v3/pull/125 and https://github.com/tc39/proposal-intl-numberformat-v3/pull/132 .

Reply:
I'll resolve this because it was discussed in the previous issue @gibson042 linked above.